tox.ini

```diff
@@ -1,13 +1,14 @@
 [tox]
-envlist = py38, py39, py310
+#envlist = py38, py39, py310
```

Comment:
Any reasoning behind commenting out vs. deleting?

Reply:
No reason just didn't notice it 😅